tests/test_util/aiohttp.py

```diff
@@ -270,3 +292,31 @@ def create_session(hass, *args):
         side_effect=create_session,
     ):
         yield mocker
+
+
+class MockLongPollSideEffect:
+    """Imitate a long_poll request. Once created, actual responses are queued and if queue is empty, will await until done."""
```

Comment:
The first sentence of a docstring should fit on one line and is a header. Then a blank line should follow and the body of the docstring should come. The body can be multiple lines. Break long lines.

See PEP257 for more info:
https://www.python.org/dev/peps/pep-0257/

Reply:
Indeed, strange that the tests pass. Does flake8 still run?
@ziv1234 maybe fix it in #33939?

Reply:
either way, maybe a separate PR since this doesn't have anything to do with qwikswitch but makes a change (albeit a doc change only) in the core test infra

Reply:
fixed in PR #34024